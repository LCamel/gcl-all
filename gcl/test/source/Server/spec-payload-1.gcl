[!
skip
!]

[!
    skip
!]


[!
    skip
        skip
    abort
!]


[!skip!]


[!  skip  !]

if True ->  [!
            ?
            !]
fi

{ True }