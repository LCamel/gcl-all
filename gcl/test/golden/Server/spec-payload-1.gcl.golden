### Result

[["skip"], ["skip"], ["skip", "    skip", "abort"], ["skip"], ["skip"], ["?"]]

### Source

[!
skip
!]

[!
    skip
!]


[!
    skip
        skip
    abort
!]


[!skip!]


[!  skip  !]

if True ->  [!
            ?
            !]
fi

{ True }

### Trace

TraceLog "      [ save ] Parsed"
TraceLog "      [ save ] Converted"
TraceLog "      [ save ] TypeChecked"
TraceLog "      [ save ] Swept"
